API formatting (#3766)
SumoLogic/sumologic-documentation: docs/api/troubleshooting.md

The [Search Job API documentation](/docs/api/search-job) provides an example of how to use the API (and does specify that cookies are to be supplied). Better error messaging around this type of error is under review and should be provided in a future release.

## Search Job API results into formatted JSON file

> When getting current [Search Job API](/docs/api/search-job) results, the results can be nicely formatted into a JSON file.

Author: ```suggestion
When getting current [Search Job API](/docs/api/search-job) results, you can format them cleanly into a JSON file.
```
